src/components/GroupButton/index.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { RemoveCircle, AddCircleOutline } from "@material-ui/icons";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+
+class GroupedButtons extends React.Component {
+  state = { counter: 0 };
+
+  handleIncrement = () => {
+    this.setState((state) => ({ counter: state.counter + 1 }));
+  };
+
+  handleDecrement = () => {
+    this.setState((state) => ({ counter: state.counter - 1 }));
+  };
+  render() {
+    const displayCounter = this.state.counter > 0;
+
+    return (
+      <div>
+        <TextField
+          id="guest"
+          disabled
+          label="Guest"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          variant="outlined"
+          style={{ position: "relative" ,
+          
+        }}
+        ></TextField>
+        <RemoveCircle
+          onClick={this.handleDecrement}
+          style={{
+            position: "absolute",
+            left: "28.5rem",
+            top: "1rem",
+            color: "rgba(233, 30, 30, 0.7)",
+            cursor: "pointer",
+            '@media (max-width:935px)': {
+              left: "-2rem",
+              top: "1rem",
+            },
```

Comment:
add some size to the left so that the remove button appear at the start of the input just to make value between increment button and decrement button